src/kit/internal/functions.ts

```diff
@@ -365,15 +365,16 @@ export const humanReadableNumber = (num: number, precision = DEFAULT_PRECISION):
     content = 'NaN';
   } else if (!Number.isFinite(num)) {
     content = `${num < 0 ? '-' : ''}Infinity`;
-  } else if (!Number.isInteger(num)) {
-    content = num.toFixed(Math.max(precision, 0));
+  } else {
+    if (!Number.isInteger(num)) {
+      content = num.toFixed(Math.max(precision, 0));
+    }
```

Comment:
its a bit confusing to have `if` here cuz of the following code. 
Could you add this code in else if? 

```suggestion
  } else if (!Number.isInteger(num)) {
    content = num.toFixed(Math.max(precision, 0));
  } else {
```